latch_cli/services/init/common/.dockerignore

```diff
@@ -3,7 +3,7 @@ README.md
 CHANGELOG.md
 docker-compose.yml
 Dockerfile
-.latch
+.latch/.logs
```

Comment:
del, we don't wanna copy over things like ssh keys or whatever else we store in `.latch`

Reply:
if you want to explicitly only include the jit entrypoint do something like this: https://stackoverflow.com/a/51429852